src/TropicalGeometry/hypersurface.jl

```diff
@@ -60,7 +60,7 @@ end
 @doc Markdown.doc"""
     TropicalHypersurface(f::AbstractAlgebra.Generic.MPoly{Oscar.TropicalSemiringElem{T}})
 
-Return the tropical hypersurface defined by a tropical polynomial.
+Returns the tropical hypersurface of a tropical polynomial `f`.
```

Comment:
We prefer imperative form, following [the Julia conventions for this](https://docs.julialang.org/en/v1/manual/documentation/)
```suggestion
Return the tropical hypersurface of a tropical polynomial `f`.
```

Reply:
(Obviously this applies to the rest of this PR, too)

Reply:
Thanks for pointing this out.